src/grimp/application/usecases.py

```diff
@@ -30,6 +31,7 @@ def build_graph(
         - package_name: the name of the top level package for which to build the graph.
         - additional_package_names: tuple of the
         - include_external_packages: whether to include any external packages in the graph.
+        - include_type_checking_imports: whether to include imports made in type checking guards.
```

Comment:
We need to add this to the docs too. I'm happy to do this if you don't feel confident.

Reply:
Yep I still need to update docs and test, see the checkboxes as well

Reply:
Updated the doc for build_graph